src/app/templates/partials/metadata-tab.ftl

```diff
@@ -18,27 +19,48 @@
             ${elapsedString}
         </div>
     </div>
-    <#if report.gitBranch?? || report.gitCommit??>
+    <#if report.gitBranch?? || report.gitCommit?? || instances?has_content>
         <hr id="git-hr"/>
-    </#if>
-    <#if report.gitBranch??>
-        <div id="git-branch-row" class="row">
-            <div id="git-branch-label" class="col-2 text-right">
-                Git branch:
-            </div>
-            <div id="git-branch-value" class="col-4">
-                ${report.gitBranch}
-            </div>
-        </div>
-    </#if>
-    <#if report.gitCommit??>
-        <div id="git-commit-row" class="row">
-            <div id="git-commit-label" class="col-2 text-right">
-                Git commit:
+        <div class="row">
+        <#if report.gitBranch?? || report.gitCommit??>
+            <div class="col">
+                <#if report.gitBranch??>
+                    <div id="git-branch-row" class="row">
+                        <div id="git-branch-label" class="col-4 text-right">
+                            Git branch:
+                        </div>
+                        <div id="git-branch-value" class="col-8">
+                            ${report.gitBranch}
+                        </div>
+                    </div>
+                </#if>
+                <#if report.gitCommit??>
+                    <div id="git-commit-row" class="row">
+                        <div id="git-commit-label" class="col-4 text-right">
+                            Git commit:
+                        </div>
+                        <div id="git-commit-value" class="col-8">
+                            ${report.gitCommit}
+                        </div>
+                    </div>
+                </#if>
             </div>
-            <div id="git-commit-value" class="col-4">
-                ${report.gitCommit}
+        </#if>
+            <div class="col">
+                <#list instances as instanceKey, instanceValue>
+                    <div class="row db-instance-row">
+                        <div class="col-4 text-right db-instance-label">
+                            Database "${instanceKey}":
+                        </div>
+                        <div class="col-8 db-instance-value">
+                            ${instanceValue}
+                        </div>
+                    </div>
+                </#list>
             </div>
+            <#if !report.gitBranch?? && !report.gitCommit??>
+                <div class="col"></div>
+            </#if>
```

Comment:
This just adds a phantom column to the right of the instances column in the unlikely event that it renders but git branches and commit do not.

Reply:
This is so that the text aligns with the row above? 
Maybe it would be better to always keep instances as the second column, so that it doesn't shift around between versions, but have the first column empty if no git values? That would just be a case of removing the conditional defined at line 25.